pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: bitcount
-version: 0.1.2
+version: 0.1.3
 description: >-
   Provides efficient one-bits count method extension for default int.
 homepage: https://github.com/gagoman/bitcount
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
Do we really need to update min version of sdk? I mean, `toInt` is available in `2.6`, right?

Reply:
Ok, I've read a little bit about migration here (https://dart.dev/null-safety) and it seems like we need to bump not only min sdk version but dependencies as well.

After:

```
$ dart pub upgrade --null-safety
```

and

```
$ dart migrate --apply-changes
```

I also have changes in `pubspec.yaml`:

```
diff --git a/pubspec.yaml b/pubspec.yaml
index 542552b..86c6611 100644
--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -5,9 +5,9 @@ description: >-
 homepage: https://github.com/gagoman/bitcount
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
 
 dev_dependencies:
-  test: ^1.0.0
-  pedantic: ^1.9.0
-  benchmark_harness: ^1.0.0
+  test: ^1.16.8
+  pedantic: ^1.11.0
+  benchmark_harness: ^2.0.0
```

Please double check and update PR.